tsconfig.json

```diff
@@ -5,9 +5,15 @@
     "esModuleInterop": false,
     "sourceMap": true,
     "noUnusedLocals": true,
+    "allowJs": true,
```

Comment:
Personally I don't like this enabling this. Ideally the `src/types/index.js` file should be converted to typescript but my lack of experience let me down. Because this is now enabled, I also had to explicitly exclude some of the other `.js` file.